cmd/vault-subpath-proxy/main.go

```diff
@@ -329,15 +330,15 @@ func startLoadingKubeconfigs(explicitPath, kubeconfigDir string) (func() map[str
 	}, nil
 }
 
-func loadKubeconfigs(explicitPath, kubeconfigDir string, callBack func(fsnotify.Event)) (map[string]ctrlruntimeclient.Client, error) {
-	kubeconfigs, err := util.LoadKubeConfigs(explicitPath, kubeconfigDir, callBack)
+func loadKubeconfigs(kubernetesOptions flagutil.KubernetesOptions, callBack func()) (map[string]ctrlruntimeclient.Client, error) {
+	kubeconfigs, err := kubernetesOptions.LoadClusterConfigs(callBack)
 	if err != nil {
 		return nil, err
 	}
 
 	clients := map[string]ctrlruntimeclient.Client{}
 	for cluster, config := range kubeconfigs {
-		client, err := ctrlruntimeclient.New(config, ctrlruntimeclient.Options{})
+		client, err := ctrlruntimeclient.New(&config, ctrlruntimeclient.Options{})
```

Comment:
you can not pass on a pointer to the loop variable here

Reply:
Oh. I made similar mistake before.